lib/galaxy/webapps/galaxy/fast_app.py

```diff
@@ -138,3 +138,10 @@ def initialize_fast_app(gx_wsgi_webapp, gx_app):
         parent_app.mount(gx_app.config.galaxy_url_prefix, app=app)
         return parent_app
     return app
+
+
+__all__ = (
+    "add_galaxy_middleware",
+    "add_request_id_middleware",
```

Comment:
Re-exported, imported from here in `test/unit/webapps/test_request_scoped_sqlalchemy_sessions.py`